pkg/executor/executortype/newdeploy/newdeploy.go

```diff
@@ -136,7 +136,7 @@ func (deploy *NewDeploy) getDeploymentSpec(ctx context.Context, fn *fv1.Function
 	}
 
 	gracePeriodSeconds := int64(6 * 60)
-	if env.Spec.TerminationGracePeriod > 0 {
+	if !(env.Spec.TerminationGracePeriod < 0) {
```

Comment:
Should we write it as `env.Spec.TerminationGracePeriod >= 0` ? same for other condition

Reply:
Ok